packages/libs/src/sdk/api/generated/default/apis/TracksApi.ts

```diff
@@ -637,6 +638,10 @@ export class TracksApi extends runtime.BaseAPI {
             queryParameters['limit'] = params.limit;
         }
 
+        if (params.userId !== undefined) {
```

Comment:
just calling out that we should probably use `currentUserId` all the way through the stack?